app/metrics.yaml

```diff
@@ -6176,3 +6176,31 @@ search_terms:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-11-01"
+  group_size_distribution:
+    type: custom_distribution
+    description: |
+      The distribution of search term tab group sizes. Rather than reporting
```

Comment:
Brilliant, this description helped make it easy to understand what you're doing in the code. :)